api/src/main/resources/hibernate.cfg.xml

```diff
@@ -10,112 +10,101 @@
-		<mapping resource="org/openmrs/api/db/hibernate/ConceptProposal.hbm.xml" />
-		<mapping resource="org/openmrs/api/db/hibernate/ConceptStateConversion.hbm.xml" />
-		<mapping resource="org/openmrs/api/db/hibernate/ConceptSet.hbm.xml" />
-		<mapping resource="org/openmrs/api/db/hibernate/ConceptMap.hbm.xml" />
-		<mapping resource="org/openmrs/api/db/hibernate/ConceptStopWord.hbm.xml" />
-		<mapping resource="org/openmrs/api/db/hibernate/ConceptSource.hbm.xml" />
-		<mapping resource="org/openmrs/api/db/hibernate/ConceptReferenceTerm.hbm.xml" />
-		<mapping resource="org/openmrs/api/db/hibernate/ConceptMapType.hbm.xml" />
-		<mapping resource="org/openmrs/api/db/hibernate/ConceptReferenceTermMap.hbm.xml" />
-		<mapping resource="org/openmrs/api/db/hibernate/Drug.hbm.xml" />
-		<mapping resource="org/openmrs/api/db/hibernate/DrugIngredient.hbm.xml" />
-		<mapping resource="org/openmrs/api/db/hibernate/DrugReferenceMap.hbm.xml" />
-		<mapping resource="org/openmrs/api/db/hibernate/Field.hbm.xml" />
-		<mapping resource="org/openmrs/api/db/hibernate/FieldAnswer.hbm.xml" />
-		<mapping resource="org/openmrs/api/db/hibernate/FieldType.hbm.xml" />
-		<mapping resource="org/openmrs/api/db/hibernate/Form.hbm.xml" />
-		<mapping resource="org/openmrs/api/db/hibernate/FormField.hbm.xml" />
-        <mapping resource="org/openmrs/api/db/hibernate/FormResource.hbm.xml" />
-		<mapping resource="org/openmrs/api/db/hibernate/GlobalProperty.hbm.xml" />
-		<mapping resource="org/openmrs/api/db/hibernate/Obs.hbm.xml" />
-		<mapping resource="org/openmrs/api/db/hibernate/Person.hbm.xml" />
-		<mapping resource="org/openmrs/api/db/hibernate/PersonAttribute.hbm.xml" />
-		<mapping resource="org/openmrs/api/db/hibernate/PersonAttributeType.hbm.xml" />
-		<mapping resource="org/openmrs/api/db/hibernate/PersonAddress.hbm.xml" />
-		<mapping resource="org/openmrs/api/db/hibernate/PersonMergeLog.hbm.xml" />		
-		<mapping resource="org/openmrs/api/db/hibernate/PersonName.hbm.xml" />
-		<mapping resource="org/openmrs/api/db/hibernate/User.hbm.xml" />
-		<mapping resource="org/openmrs/api/db/hibernate/LoginCredential.hbm.xml" />
-		<mapping resource="org/openmrs/api/db/hibernate/Privilege.hbm.xml" />
-		<mapping resource="org/openmrs/api/db/hibernate/Role.hbm.xml" />
-		<mapping resource="org/openmrs/api/db/hibernate/Patient.hbm.xml" />
-		<mapping resource="org/openmrs/api/db/hibernate/PatientIdentifier.hbm.xml" />
-		<mapping resource="org/openmrs/api/db/hibernate/PatientIdentifierType.hbm.xml" />
-		<mapping resource="org/openmrs/api/db/hibernate/Relationship.hbm.xml" />
-		<mapping resource="org/openmrs/api/db/hibernate/RelationshipType.hbm.xml" />
-		<mapping resource="org/openmrs/api/db/hibernate/OrderType.hbm.xml" />
-		<mapping resource="org/openmrs/api/db/hibernate/Order.hbm.xml" />
-		<mapping resource="org/openmrs/api/db/hibernate/OrderSet.hbm.xml" />
-		<mapping resource="org/openmrs/api/db/hibernate/OrderSetMember.hbm.xml" />
```

Comment:
It would be great if your do not change any formatting for this file. That way, it will show only what actually changed.

Reply:
I'll restore the default format, editor auto-formatted before I turned it off